src/Controller/UserProfileController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2018 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+namespace App\Controller;
+
+use BEdita\SDK\BEditaClientException;
+use Cake\Core\Configure;
+use Cake\Event\Event;
+use Cake\Http\Response;
+use Psr\Log\LogLevel;
+
+/**
+ * User Profile controller: view and edit logged user data
+ *
+ * @property \App\Controller\Component\PropertiesComponent $Properties
+ */
+class UserProfileController extends AppController
+{
+    /**
+     * View profile data
+     *
+     * @return void
+     */
+    public function view() : void
+    {
+        $this->request->allowMethod(['get']);
+
+        $id = $this->Auth->user('id');
+        try {
+            $response = $this->apiClient->getObject($id, 'users');
```

Comment:
Here we should use `GET /auth/user`
https://docs.bedita.net/en/latest/endpoints/auth.html#who-am-i

`/users` endpoint may not be accessibile or read-only